File: rmt/src/test/scala/de/flower/rmt/ui/page/users/manager/PlayersPageTest.scala
package de.flower.rmt.ui.page.users.manager

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class PlayersPageTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startPage(new PlayersPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/page/users/manager/PlayersPageTest.java
package de.flower.rmt.ui.page.users.manager;

import de.flower.rmt.test.AbstractRMTWicketIntegrationTests;
import org.testng.annotations.Test;

/**
 * @author flowerrrr
 */

public class PlayersPageTest extends AbstractRMTWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new PlayersPage());
        wicketTester.dumpPage();
    }
}